test-packages/test-utils/src/utils/download-ui5-resources.ts

```diff
@@ -0,0 +1,78 @@
+import { TestModelVersion } from "../../api";
+import { reduce, keys, map } from "lodash";
+import { resolve } from "path";
+import { writeFile, mkdirs, pathExists } from "fs-extra";
+import fetch from "node-fetch";
+
+async function getLibs(version: TestModelVersion): Promise<string[]> {
+  // The metadata.json seems to have been added only very recently :(
+  let versionInURL: string = version;
```

Comment:
The name `versionInURL` could be more clear if it indicated it is used in **metadata** url.
As I am sure more URLs are built somewhere along this flow.

Also we should explicitly document the assumption that the list of libraries has not changed, (maybe even check with UI5 team?)

Reply:
I'm pretty sure it has changed since some of the URLs return an error for older versions. Hopefully they didn't remove any libraries but only added, but yes, we should check with them.